ngrok/src/config/tcp.rs

```diff
@@ -116,6 +118,11 @@ impl TcpTunnelBuilder {
         self.options.common_opts.metadata = Some(metadata.into());
         self
     }
+    /// Sets the ingress configuration for this endpoint
+    pub fn bindings(&mut self, bindings: impl Into<String>) -> &mut Self {
```

Comment:
It looks like the plurality here is wrong since this is adding a single binding.
The other build-a-map-or-vec-one-item-at-a-time builders are also singular, i.e.

```rust
    let tunnel = sess
        .http_endpoint()
        .allow_cidr("1.2.3.4/0")
        .allow_cidr("2.3.4.5/0")
        .request_header("a", "b")
        .request_header("b", "b")
```

so it feels like it should also be:

```rust
        .binding("public")
        .binding("foo")
```